firefox-ios/Client/Frontend/Browser/BrowserViewController/Actions/GeneralBrowserAction.swift

```diff
@@ -57,7 +58,7 @@ enum GeneralBrowserActionType: ActionType {
     case showNewTabLongPressActions
     case addToReadingListLongPressAction
     case clearData
-    case showPasswordGenerator
+    case showPasswordGenerator(frame: WKFrameInfo)
```

Comment:
This should be a class variable instead of an associated value.

Reply:
@DanielDervishi r+ after above change, can re-review it once ready